synapse/server.py

```diff
@@ -235,6 +235,7 @@ def __init__(self, hostname: str, config: HomeServerConfig, reactor=None, **kwar
         self.start_time = None
 
         self.instance_id = random_string(5)
+        self.instance_name = config.worker_name or "master"
```

Comment:
this should probably be underscored if it's intended that we use `get_instance_name` to access it.